README.md

```diff
@@ -13,14 +13,14 @@ zone.  This role has a trust relationship with the users account.
 1. Run the command `terraform apply`.
 
 <!-- BEGIN_TF_DOCS -->
-## Requirements ##
+## Requirements
 
 | Name | Version |
 |------|---------|
 | terraform | ~> 1.0 |
 | aws | ~> 4.9 |
 
-## Providers ##
+## Providers
```

Comment:
Revert this change that terraform-docs makes:
```suggestion
## Providers ##
```

Reply:
Fixed in https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/103/commits/5de085cd1da6a42095891d88144f4bf0b14543a4.